src/patternfly/components/Popover/popover.scss

```
@@ -47,7 +47,7 @@
   --#{$popover}__arrow--m-block-right--InsetInlineEnd: var(--pf-t--global--border--radius--medium);
 
   // Close
-  --#{$popover}__close--InsetBlockStart: calc(var(--#{$popover}__content--PaddingBlockStart) - (var(--pf-t--global--spacer--control--vertical--compact) * 1.5)); // align top of button with top of text
+  --#{$popover}__close--InsetBlockStart: calc(var(--#{$popover}__content--PaddingBlockStart) - var(--pf-t--global--spacer--action--vertical--plain)); // align top of button with top of text 
```

From the last meeting we had with @thatblindgeye and @lboehling, we're going to use `--pf-t--global--spacer--control--vertical--[...]` for the top/bottom padding of all buttons/controls, so since this var references the top padding of an action, I'd update this to

```suggestion
  --#{$popover}__close--InsetBlockStart: calc(var(--#{$popover}__content--PaddingBlockStart) - var(--pf-t--global--spacer--control--vertical--default)); // align top of button with top of text 
```